src/amo/components/SuggestedPages/index.js

```diff
@@ -1,33 +1,19 @@
-/* @flow */
-import config from 'config';
 import * as React from 'react';
-import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import { compose } from 'redux';
 
 import Link from 'amo/components/Link';
-import { shouldShowThemes } from 'amo/utils';
 import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
 import translate from 'core/i18n/translate';
 import { visibleAddonType } from 'core/utils';
-import type { AppState } from 'amo/store';
-import type { I18nType } from 'core/types/i18n';
 
-type Props = {||};
-
-type InternalProps = {|
-  ...Props,
-  _config: typeof config,
-  i18n: I18nType,
-  clientApp: string,
-|};
-
-export class SuggestedPagesBase extends React.Component<InternalProps> {
-  static defaultProps = {
-    _config: config,
-  };
+export class SuggestedPagesBase extends React.Component {
```

Comment:
We should keep the Flow types in this file. Did you revert the commit with `git revert` by chance? If so, that's great! But we can add a few things back :)

Reply:
I am a bit scared of `git revert` especially when its not me who wrote the code.
I referred your commit `https://github.com/mozilla/addons-frontend/pull/6857/files`